src/borg/archiver.py

```diff
@@ -2665,42 +2687,40 @@ def do_break_lock(self, args, repository):
             # note: using = is essential to avoid cmdline argument parsing issues.
             borg create --pattern=+pics/2018/good --pattern=-pics/2018 repo::arch pics
 
+            # backup only JPG/JPEG files (case insensitive) in all home directories:
+            borg create --pattern 're:\\.jpe?g(?i)' repo::arch /home
+
             # use a file with patterns:
-            borg create --patterns-from patterns.lst repo::arch
+            borg create --patterns-from patterns.lst repo::arch /
 
         The patterns.lst file could look like that::
 
-            # "sh:" pattern style is the default, so the following line is not needed:
+            # sh: pattern style is the default, so the following line is not needed:
             P sh
-            R /
-            # can be rebuild
+            # exclude caches
             - home/*/.cache
-            # they're downloads for a reason
-            - home/*/Downloads
-            # susan is a nice person
             # include susans home
             + home/susan
             # also back up this exact file
             + pf:home/bobby/specialfile.txt
             # don't backup the other home directories
             - home/*
-            # don't even look in /proc
-            ! proc
+            # don't even look in /dev, /proc, /run, /sys, /tmp
+            ! re:^(dev|proc|run|sys|tmp)/
```

Comment:
if there is a slash at the end, borg might try to look "into" these directories, because the directory was not matched by the expression (yet), I guess?

Reply:
That's right, but without / it would exclude also files like "/device". Not very likely, but...
I remove the / and add a comment with a warning.